wp-includes/class-wp-service-workers.php

```diff
@@ -63,6 +70,43 @@ public function init() {
 		do_action_ref_array( 'wp_default_service_workers', array( &$this ) );
 	}
 
+	/**
+	 * Get workbox script.
+	 *
+	 * @return string Script.
+	 */
+	public function get_workbox_script() {
+
+		// @todo The Workbox sources will probably need to be installed locally to avoid the external request(s).
+		$script = sprintf(
+			"importScripts( %s );\n",
+			wp_json_encode( 'https://storage.googleapis.com/workbox-cdn/releases/3.4.1/workbox-sw.js', 64 /* JSON_UNESCAPED_SLASHES */ )
```

Comment:
It's totally cool to prototype against the `storage.googleapis.com` hosted version of the code, but if/when you do get closer to deploying this more widely, we'd like to avoid a WordPress level of traffic for our CDN bucket 😄

Reply:
@jeffposnick Thanks. We'll be sure to `workbox copyLibraries` to copy the scripts into WordPress instead of using the CDN.